src/components/ReplayControls/ReplayControls.tsx

```diff
@@ -0,0 +1,60 @@
+import type { PlayRecorder, ReplayState } from '@game/recorder';
+
+import { ArrowRightIcon } from '@components/ArrowRightIcon';
+import { BlockIcon } from '@components/BlockIcon';
+import type { Board } from '@game/board';
+import { Move } from '../Move/Move';
+import styles from './ReplayControls.module.scss';
+
+type ReplayControlsProps = {
+  board: Board;
+  isDisabled: boolean;
+  recorder: PlayRecorder;
+  replayState: ReplayState;
+};
+
+export function ReplayControls({
+  board,
+  isDisabled,
+  recorder,
+  replayState,
+}: ReplayControlsProps) {
+  return (
+    <div className={styles['replay-controls']}>
+      <div className={styles['replay-progress-row']}>
+        <span>
+          Replay ({replayState.currentMoveIndex} /{' '}
+          {replayState.recording.moves.length})
+        </span>
+        <progress
```

Comment:
Make a progress component based on this codepen:

https://codepen.io/orjandesmet/pen/XJdVPZB